kafka_topic_enforcer/src/test/java/com/tesla/data/topic/enforcer/TopicEnforcerTest.java

```diff
@@ -95,7 +100,7 @@ public void testAlterConfiguration() {
     enforcer.alterConfiguration();
 
     // topic 'a'
-    List<ConfiguredTopic> expected = Collections.singletonList(configured.get(0));
+    List<ConfiguredTopic> expected = Arrays.asList(configured.get(0), configured.get(3));
```

Comment:
is this change needed? If yes, can you fix the comment, looks like its out of date

Reply:
updated that line because I added another test case there, another topic which should be changing (topic "d"). updated the comments to reflect it